### PR TITLE
docs: rename option of ResponseTime

### PR DESCRIPTION
**Pull request by rokamu623**
## Description
The current option names for ResponseTime diverge from the intuitive image.
The option names have been changed as follows:
- worst -> worst-with-external-latency
- worst-in-input -> worst

## Related links
Jira: https://tier4.atlassian.net/browse/RT2-888

## Pre-review checklist for the PR author

- [x] I've confirmed the [contribution guidelines](https://github.com/tier4/caret/blob/main/.github/CONTRIBUTING.md).
- [x] The PR is ready for review.

## In-review checklist for the PR reviewers

The PR reviewers **must** check the checkboxes below before approval.

- [ ] The PR has been reviewed.

## Post-review checklist for the PR author

The PR author **must** check the checkboxes below before merging.

- [ ] There are no open discussions or they are tracked via tickets.
- [x] The PR is ready for merge.

After all checkboxes are checked, anyone who has write access can merge the PR.

### CODE REVIEW

**Review by bopeng-saitama — APPROVED** [review]
LGTM

File: docs/visualization/path/response_time.md [hunk 1]
```diff
@@ -10,9 +10,9 @@ Please refer to [FAQ](../../faq/faq.md#how-response-time-is-calculated) if you a
     However, both will be referred to as "response time" for simplicity in this document.
 <prettier-ignore-end>
 
-Three cases of response time is defined; `best-to-worst`, `best`, and `worst`. With `best` case, CARET samples shortest time elapsed in a targeted path. With `worst` case, the previous message input timing is taken into account. `best-to-worst` case includes most of all cases between `best` and `worst`.
+Three cases of response time is defined; `all`, `best`, `worst`, and `worst-with-external-latency`. With `all` case, CARET samples all time elapsed in a targeted path. With `best` case, CARET samples shortest time elapsed in a targeted path. With `worst` case, CARET samples largest time elapsed in a targeted path. With `worst-with-external-latency` case, the previous message input timing is taken into account. `best-to-worst` case includes most of all cases between `best` and `worst`.
```
Comment (nabetetsu) [inline docs/visualization/path/response_time.md:13]:
```suggestion
As a visualisation of the graph, four different cases can be specified in the `case` argument. Default value is 'all'; 'all', 'best', 'worst', and 'worst-with-external-latency'.

- `all` case
  - CARET displays the elapsed time from all input times in the same cycle.
- `best` case
  - CARET displays the elapsed time that is the shortest of all input times in the same cycle.
- `worst` case
  - CARET displays the elapsed time that is the longest of all input times in the same cycle. 
- 'worst-with-external-latency` case
  - CARET displays the elapsed time from the last input time of a previous cycle.
```

File: docs/visualization/path/response_time.md [hunk 2]
```diff
@@ -10,9 +10,9 @@ Please refer to [FAQ](../../faq/faq.md#how-response-time-is-calculated) if you a
...
 
-This section shows two methods to visualize response time, `Histogram` and `Stacked Bar`.
+This section shows two methods to visualize response time, `Histogram`, `Stacked Bar` and `TimeSeries`.
```
Comment (nabetetsu) [inline docs/visualization/path/response_time.md:15]:
```suggestion
This section shows three methods to visualize response time, `Histogram`, `Stacked Bar` and `TimeSeries`.
```

File: docs/visualization/path/response_time.md [hunk 3]
```diff
@@ -45,113 +45,113 @@ plot.show()
...
 ```
 
-![response_time_worst_in_input_histogram](../imgs/response_time_histogram_worst_in_input.png)
+![response_time_histogram_worst_with_external_latency](../imgs/response_time_histogram_worst_with_external_latency.png)
 
 ## Stacked Bar
 
 The following scripts generate stacked bar graphs of response time.
-Horizontal axis of the stacked bar graph means timeseries and vertical axis means `Response time` in the worst, best, all, worst-in-input case.
+Horizontal axis of the stacked bar graph means timeseries and vertical axis means `Response time` in the all, best, worst, worst-with-external-latency case.
```
Comment (nabetetsu) [inline docs/visualization/path/response_time.md:77]: In the sample graph, `latency` is displayed, but is it the same value as response time?
```suggestion
Horizontal axis of the stacked bar graph means `system time` or `index` and vertical axis means response time.
```

File: docs/visualization/path/response_time.md [hunk 4]
```diff
@@ -45,113 +45,113 @@ plot.show()
...
+![response_time_stacked_bar_worst_with_external_latency](../imgs/response_time_stacked_bar_worst_with_external_latency.png)
 
 The horizontal axis can be changed to `system time` or `index` by changing `plot.show()` to `plot.show(xaxis_type='index')`.
 
 `[worst - best] response time` in legends refers to the difference in response time between the worst case and the best case.
 
 ## TimeSeries
 
-The following scripts generate timeseries graphs of response time. The horizontal axis means time, labeled as Time [s]. The vertical axis means `Response time` in the best case, worst case, worst-in-input case, or all case.
+The following scripts generate timeseries graphs of response time. The horizontal axis means time, labeled as Time [s]. The vertical axis means `Response time` in the all, best, worst, worst-with-external-latency case.
```
Comment (nabetetsu) [inline docs/visualization/path/response_time.md:120]:
```suggestion
The following scripts generate timeseries graphs of response time. The horizontal axis means `system time` or `index`. The vertical axis means `Response time`.
```

File: docs/visualization/path/response_time.md (hunk 1 above)
Comment (rokamu623) [reply docs/visualization/path/response_time.md]: Fixed it in [2211093](https://github.com/tier4/CARET_doc/pull/245/commits/2211093ddd6a8c037203df4f4072a5374e4bc7ab).

File: docs/visualization/path/response_time.md (hunk 2 above)
Comment (rokamu623) [reply docs/visualization/path/response_time.md]: Fixed it in [2211093](https://github.com/tier4/CARET_doc/pull/245/commits/2211093ddd6a8c037203df4f4072a5374e4bc7ab).

File: docs/visualization/path/response_time.md (hunk 4 above)
Comment (rokamu623) [reply docs/visualization/path/response_time.md]: Fixed it in [2211093](https://github.com/tier4/CARET_doc/pull/245/commits/2211093ddd6a8c037203df4f4072a5374e4bc7ab).

File: docs/visualization/path/response_time.md (hunk 3 above)
Comment (rokamu623) [reply docs/visualization/path/response_time.md]: Fixed it in [2211093](https://github.com/tier4/CARET_doc/pull/245/commits/2211093ddd6a8c037203df4f4072a5374e4bc7ab).

Is the description of the vertical axis of the stacked bar correct as per the change?

**Review by nabetetsu — APPROVED** [review]
LGTM :+1:

**Pull request merged**